src/NzbDrone.Core/Datastore/Migration/042_remove_album_folders.cs

```diff
@@ -0,0 +1,18 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(42)]
+    public class remove_album_folders : NzbDroneMigrationBase
+    {
+        protected override void MainDbUpgrade()
+        {
+            Delete.Column("AlbumFolder").FromTable("Artists");
+
+            Execute.Sql("UPDATE NamingConfig SET StandardTrackFormat = AlbumFolderFormat || '/' || StandardTrackFormat, MultiDiscTrackFormat = AlbumFolderFormat || '/' || MultiDiscTrackFormat");
```

Comment:
Should we be a bit smarter here and not update if user has disabled album folders for all artists?

Reply:
Does it work on windows with / too?

Reply:
Yea, the name builder splits on either and recombines using path.combine